src/main/twirl/gitbucket/core/main.scala.html

```diff
@@ -4,7 +4,9 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Both this line and 2 lines down are needed?

Reply:
it removes the zoom/scalling on both mobile and tablet when <input type="text"... is hit on mobile it makes it more compatible on these platforms

Reply:
I mean there are this line
```html
<meta http-equiv="X-UA-Compatible" content="IE=edge">
```
and
```html
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
```
Are you pointing them?

Reply:
*If you are using the X-UA-Compatible META tag you want to place it as close to the top of the page's HEAD as possible. Internet Explorer begins interpreting markup using the latest version. When Internet Explorer encounters the X-UA-Compatible META tag it starts over using the designated version's engine. This is a performance hit because the browser must stop and restart analyzing the content.*

This will help anything with Microsoft edge browser engine like windows tablet, phone and xbox etc etc Like i said before, these line of codes will help with the overall handling of the website on each available platform on the market with the proper bootstrap scaling :)

Reply:
> When Internet Explorer encounters the X-UA-Compatible META tag it starts over using the designated version's engine. This is a performance hit because the browser must stop and restart analyzing the content.

So there's a reason to have browser re-start analyzing twice with TWO exactly the same meta tags?

IMHO remove the latter X-UA-Compatible's meta tag so it doesn't need to restart twice, only once. (aka remove `<meta http-equiv="X-UA-Compatible" content="IE=edge" />`)

Reply:
> IMHO remove the latter X-UA-Compatible's meta tag so it doesn't need to restart twice, only once. (aka remove <meta http-equiv="X-UA-Compatible" content="IE=edge" />)

Thanks! This is what I wanted to say.